pkg/api/v1alpha1/cluster.go

```diff
@@ -856,3 +858,26 @@ func validatePackageControllerConfiguration(clusterConfig *Cluster) error {
 	}
 	return nil
 }
+
+func validateCloudstackK8sVersion(cluster *Cluster) error {
```

Comment:
Where is this called? It feels a little unnecessary to wrap it in a function.

Reply:
It's called in cluster webhook. https://github.com/aws/eks-anywhere/pull/5737/files#diff-cae2b8e151554f7b5daba7a32c010865bd6718ee6c3afd6345cafe71e8814521R205

Reply:
Thanks.